src/main.cpp

```diff
@@ -349,6 +349,22 @@ int main(int argc, char *argv[])
                FILENAMES["user_dir"].c_str());
         exit(1);
     }
+    if (!assure_dir_exist(FILENAMES["user_datadir"].c_str())) {
```

Comment:
It makes it easier for the end user to find where to install mods (rather than the extra step of the end user creating the directories himself).

I'll change that to pass a std::string.